tensorflow/python/kernel_tests/matmul_op_test.py

```diff
@@ -108,10 +160,59 @@ def Test(self):
 
   return Test
 
-
 class MatMulGradientTest(test_lib.TestCase):
-  pass  # Will be filled in below.
-
+  def test_Gradient_bfloat16(self):
```

Comment:
Same comment. We can modify the template function below for bf16-specific actions:
https://github.com/tensorflow/tensorflow/blob/b2ed6eb035c841df8c78eef183616d8cd374c9fb/tensorflow/python/kernel_tests/matmul_op_test.py#L116

Reply:
I have updated the code according to your suggestion.